Mixmatch/open_image_dataset/buildOIDv6_dataset.sh

```diff
@@ -0,0 +1,4 @@
+for seed in 1 2 3 4 5; do
+    CUDA_VISIBLE_DEVICES= ../scripts/create_split.py --seed=$seed --size=1000 $ML_DATA/SSL/OIDv6 $ML_DATA/OIDv6-train.tfrecord
+    wait
+done
```

Comment:
In the future you can use "git mv path/to/oldfile path/to/newfile" to move or rename files in git so that they are not shown as two separate changes. It's fine for this PR.

Reply:
Great. Nice to know.